composer.json

```diff
@@ -19,7 +19,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6",
```

Comment:
atm your lib requires ecc in 0.4.0 which requires 5.6 [1]. is there any api or particular feature of ecc which is required for this lib, or could we also use it with `^0.3` (which would work with php5.4 then)

[1] https://packagist.org/packages/mdanter/ecc#v0.4.0